### ISSUE TITLE
No longer working for android 12

### ISSUE THREAD

**Issue by tommyzat** [sdex/AndroidIDeditor#8]
Says root access not granted

**Comment by dizietsma** [comment]
Yup, just installed on a fresh lineageos 19.1 with magisk25, magisk says the app has superuser but when launching says that it failed to get root permission

**Comment by mrrakshi** [comment]
Is there an app to replace it?

**Comment by mrrakshi** [comment]
> Says root access not granted

 hi bro Is there an app to replace it?

**Comment by sdex** [comment]
> Is there an app to replace it?

Did you try to use any text editor that supports root permission? 
Having a separate app to edit the ids it's convenient but not really necessary. 
I don't have a rooted android 12+ device to fix the issue.

**Comment by tommyzat** [comment]
> > Is there an app to replace it?
> 
> Did you try to use any text editor that supports root permission? 
> Having a separate app to edit the ids it's convenient but not really necessary. 
> I don't have a rooted android 12+ device to fix the issue. 

Please don't try it. I bricked my phone this way

They seem to encrypt the ids or store them in a different way

**Comment by sdex** [comment]
There are significant changes in the way that the system stores IDs in Android 12. 
I'm working on the second version of the app, which supports Android 12 and above.
More information you can find in this repository:
https://github.com/sdex/AndroidIDeditorV2
